bin/storm.py

```diff
@@ -754,18 +827,24 @@ def parse_config_opts(args):
     elif token == "--config":
       global CONFFILE
       CONFFILE = curr.pop()
+    elif token == "--jars":
+      jars_list.extend(curr.pop().split(','))
+    elif token == "--artifacts":
```

Comment:
This is already documented for `jar` so you can refer `help jar` to see explanation. I've also documented to doc page.

The intention behind picking `artifacts` is that, I picked `packages` first since this is what Spark already used, but realized why they use `packages` as term is that they maintain spark-packages. So `artifacts` is the alternatives for me.
Btw, the term `artifact` is used by Maven and Ivy, and Aether also uses `Artifact` class to represent this.